src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static final Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```

Reply:
why i could not use static here?

Reply:
all question should be asked at the Mate platform